src/machine.rs

```diff
@@ -191,6 +191,7 @@ impl<'m> Machine<'m> {
         }
         trace!("{vm_id}: Running command: {:?}", cmd);
         let child = cmd.spawn()?;
+        let pid = child.id().expect("firecracker process exited early");
```

Comment:
Failure here shouldn't panic I think and doesn't seem too unlikely to happen. Maybe add a new Error variant for this case?

Reply:
We probably should also include the return value of the process, which we can get using `Child::wait` method if this is `None`.

Reply:
> done

:+1: 

pro-tip: you can just click "Resolve conversation".